mkosi/__init__.py

```diff
@@ -5680,13 +5685,27 @@ def build_auxiliary_output_path(args: argparse.Namespace, suffix: str, can_compr
     return output.with_name(f"{output.name}{suffix}{compression or ''}")
 
 
-def check_valid_script(path: Path) -> None:
+DISABLED = Path('DISABLED')  # a placeholder value to suppress autodetection
```

Comment:
hmmm, but this is a theoretically valid path, no? I mean, why not just use `/dev/null` here? I think this makes conceptionally a lot more sense, and is also a special, valid path here.

Reply:
See the discussion above. `--script=DISABLED` would actually work (and run `./DISABLED`).

Reply:
Let's add this as a comment with the constant maybe?

Reply:
>  I mean, why not just use `/dev/null`

We *could*, but it doesn't seem better. It might trick the reader into thinking that we actually read `/dev/null` and then do nothing because it's empty.

> Let's add this as a comment with the constant maybe?

Added now.